src/main/resources/requests/subscriptionRequests.ts

```diff
@@ -33,4 +34,15 @@ export class SubscriptionRequests {
     }
     return null;
   }
+
+  public async subscribe(searchResult: Array<CaseSubscription>, user): Promise<boolean> {
```

Comment:
Since this function doesn't do anything at the moment use this shorthand.
return !!(user); // this returns boolean